product_brand/__openerp__.py

```diff
@@ -0,0 +1,58 @@
+# -*- encoding: utf-8 -*-
+###############################################################################
+# #                                                                           #
+# product_brand for Odoo #                                                    #
+# Copyright (C) 2009 NetAndCo (<http://www.netandco.net>). #                  #
+# Copyright (C) 2011 Akretion Benoît Guillot <benoit.guillot@akretion.com> #  #
+# Copyright (C) 2014 prisnet.ch Seraphine Lantible <s.lantible@gmail.com> #   #
+# Contributors                                                                #
+# Mathieu Lemercier, mathieu@netandco.net, #                                  #
+# Franck Bret, franck@netandco.net #                                          #
+# Seraphine Lantible, s.lantible@gmail.com, http://www.prisnet.ch             #
+# #                                                                           # 
```

Comment:
thanks, yannick. 

Seems my problem is that I do not read the Travis reports. Actually I don't know where to find the details.  All I ever see is '...has failed checks'. Seems to me Travis gives you much more details but I just don't see where those are shown.

Now, with the removed qhitespace 1 of 3 checks did not fail. The total message still ist '... has failed checks' ... I guess you can make sense to that. Whether you have a minute to explain I'll promise to use this knowledge to help on OCA projects